src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,65 @@
+package duke;
+
+import java.nio.file.Path;
+
+import duke.command.Command;
+import duke.gui.GuiApplication;
+import duke.task.TaskList;
+import javafx.application.Application;
```

Comment:
I think making the ordering of import statements would help enhance the readability of the code.

Reply:
thank you for the feedback! actually `import` statements aren't exactly the most well-defined in the CS2103T coding standards, so i just followed the order recommended by checkstyle:

- static imports
- `java.*` imports, followed by
- other imports (including `duke`/`javafx`)

but will check the files again and see if there are any coding standard violations for `import`s i may have missed out! 👍